MaxMind.GeoIP2.UnitTests/DeserializationTests.cs

```diff
@@ -18,13 +18,15 @@ private void CanDeserializeCountryResponse(CountryResponse resp)
             Assert.Equal("North America", resp.Continent.Name);
 
             Assert.Equal(1, resp.Country.GeoNameId);
+            Assert.False(resp.Country.IsInEuropeanUnion);
             Assert.Equal("US", resp.Country.IsoCode);
             Assert.Equal(56, resp.Country.Confidence);
             Assert.Equal("United States", resp.Country.Name);
 
             Assert.Equal(2, resp.RegisteredCountry.GeoNameId);
-            Assert.Equal("CA", resp.RegisteredCountry.IsoCode);
-            Assert.Equal("Canada", resp.RegisteredCountry.Name);
+            Assert.True(resp.RegisteredCountry.IsInEuropeanUnion);
+            Assert.Equal("DE", resp.RegisteredCountry.IsoCode);
+            Assert.Equal("Germany", resp.RegisteredCountry.Name);
 
             Assert.Equal(4, resp.RepresentedCountry.GeoNameId);
             Assert.Equal("GB", resp.RepresentedCountry.IsoCode);

```

Comment:
We could have an `Assert.False()` on `RepresentedCountry.IsInEuropeanUnion` perhaps? Likewise in other tests here.